docs/.docusaurus/client-modules.js

```diff
@@ -0,0 +1,5 @@
+export default [
```

Comment:
you can ignore all the `docs` files, this is all just docusaurus boilerplate. I included it here just to confirm everything works. The next pr will clean up this folder and put in the docs content i wrote